e2e/test/iothub/method/MethodE2ETests.cs

```diff
@@ -133,7 +134,7 @@ public async Task Method_ServiceInvokeDeviceMethodWithUnknownDeviceThrows()
             try
             {
                 // Invoke the direct method asynchronously and get the response from the simulated device.
-                await serviceClient.InvokeDeviceMethodAsync("MyDummyDevice", methodInvocation);
+                await serviceClient.InvokeDeviceMethodAsync("MyDummyDevice", methodInvocation).ConfigureAwait(false);
```

Comment:
- I know you didn't add this, but maybe change `MyDummyDevice` to something else? 😄 
- We don't really need to `ConfigureAwait()` in a .netcoreapp/ .netfm application. The application's synchronization context would handle the task scheduling.